combirepo/rpm_patcher.py

```diff
@@ -660,6 +662,16 @@ def __process_results(self):
             hidden_subprocess.call("Copying to repo",
                                    ["sudo", "cp", path, target])
 
+    def __umount_root(self):
+        """
+        Processes final results of patcher.
+        """
+        kickstart_file = KickstartFile(self.kickstart_file_path)
+        images_dict = kickstart_file.get_images_mount_points()
```

Comment:
Are you sure that receiving mount points from kickstart file is the best solution?
I think that reading `/proc/self/mounts` and filtering by `root` is better idea: we can be sure that we unmounted all mount points used.